contents/docs/cdp/build/index.md

```diff
@@ -75,10 +75,10 @@ function composeWebhook(event) {
 
 ## Next steps
 
-That's all for the crash course. There's a lot you can do with connectors such as sending events elsewhere via webhooks or modifying events before they're stored. Here are some additional resources to help you get started in building your own connector for PostHog:
+That's all for the crash course. There's a lot you can do with transformation and destination such as sending events elsewhere via webhooks or modifying events before they're stored. Here are some additional resources to help you get started in building your own for PostHog:
```

Comment:
Each plugin is only 1 of this: either a transformation (`processEvent` plugin) or a destination (`composeWebhook` plugin)
```suggestion
That's all for the crash course. There's a lot you can do with transformation or destination such as sending events elsewhere via webhooks or modifying events before they're stored. Here are some additional resources to help you get started in building your own for PostHog:
```